app/controllers/application_controller.rb

```diff
@@ -14,4 +14,10 @@ def default_url_options(options = {})
   def set_locale
     I18n.locale = params[:locale] || I18n.default_locale
   end
+
+  private
```

Comment:
Can we change this to `protected` visibility. This better suites this scenario and signals this is potentially overridden in any subclasses. Also, something slightly odd is going on here as the following ruby would demonstrate:

```
irb(main):001:0> class Thing
irb(main):002:1> private
irb(main):003:1> def meh
irb(main):004:2> puts 'meh in thing'
irb(main):005:2> end
irb(main):006:1> end
=> :meh
irb(main):007:0> class Thang < Thing
irb(main):008:1> end
=> nil
irb(main):009:0> Thang.new.meh
NoMethodError: private method `meh' called for #<Thang:0x007fc9f49ce990>
        from (irb):9
        from /Users/benlovell/.rbenv/versions/2.2.0/bin/irb:11:in `<main>'
```